src/schema/contentPreference.ts

```diff
@@ -119,6 +123,10 @@ export const typeDefs = /* GraphQL */ `
       Paginate first
       """
       first: Int
+      """
```

Comment:
I'm not sure if here it should be a option @capJavert ?
Because technically can only ever be my feed right?

Reply:
User following yes, because yo can follow source, tags on different feeds. `userFollowers` no, because that only exists for main feed.

Reply:
Also if we can add a test for this

Reply:
Oh right we query it for the custom feed yeah
I added now feedPlus, but we probably need to merge these both to add tests neatly.